php-cli/Dockerfile

```
@@ -75,34 +75,38 @@ RUN apt-get update \
     && pecl install \
         mcrypt \
         oauth-2.0.3 \
+        pcov \
         redis \
         ssh2-1.2 \
         xdebug \
     && docker-php-ext-enable \
         mcrypt \
         oauth \
+        pcov \
         redis \
-        ssh2 \
-        xdebug
+        ssh2
 
 ENV COMPOSER_ALLOW_SUPERUSER 1
 ENV COMPOSER_HOME /tmp
-ENV PHP_CONFDIR=/usr/local/etc/php/conf.d
-ENV PHP_CONFIG_REGISTER_GLOBALS=Off
-ENV PHP_CONFIG_ALLOW_URL_FOPEN=On
-ENV PHP_CONFIG_ALLOW_URL_INCLUDE=Off
-ENV PHP_CONFIG_MAGIC_QUOTES_GPC=On
-ENV PHP_CONFIG_EXPOSE_PHP=On
-ENV PHP_CONFIG_MBSTRING_FUNC_OVERLOAD=0
-ENV PHP_CONFIG_MEMORY_LIMIT=-1
-ENV PHP_CONFIG_MAX_EXECUTION_TIME=0
-ENV PHP_CONFIG_UPLOAD_MAX_FILESIZE=2M
-ENV PHP_CONFIG_MAX_FILE_UPLOADS=20
-ENV PHP_CONFIG_MAX_INPUT_VARS=1000
-ENV XDEBUG_IDE_KEY=docker
-ENV XDEBUG_REMOTE_HOST=localhost
-ENV XDEBUG_REMOTE_PORT=9000
-ENV XDEBUG_PROFILER_ENABLE=Off
+ENV PCOV_DIRECTORY auto
+ENV PCOV_ENABLE Off
```

```suggestion
ENV PCOV_ENABLE 0
```

`pcov.enabled` must be an integer